File: Programmers/2016년.cpp
#include <bits/stdc++.h>
using namespace std;


string solution(int a, int b) {
    string weekend[7]={ "THU","FRI", "SAT","SUN", "MON", "TUE", "WED"};
    int day[13]={0,31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};

    string answer = "";

    int sum=0;

    for(int i=0;i<a;i++){
        sum+=day[i];
    }

    sum+=b;

    return weekend[sum%7];
}

File: Programmers/2016년.py
def solution(a, b):
    day = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    day = day * 55
    pattern = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]

    answer = day[sum(pattern[:a]) + b - 1]
    return answer